src/main/java/com/redhat/devtools/intellij/quarkus/lsp4ij/console/explorer/LanguageServerExplorerLifecycleListener.java

```diff
@@ -0,0 +1,171 @@
+package com.redhat.devtools.intellij.quarkus.lsp4ij.console.explorer;
+
+import com.redhat.devtools.intellij.quarkus.lsp4ij.LanguageServerWrapper;
+import com.redhat.devtools.intellij.quarkus.lsp4ij.lifecycle.LanguageServerLifecycleListener;
+import com.redhat.devtools.intellij.quarkus.lsp4ij.settings.ServerTrace;
+import com.redhat.devtools.intellij.quarkus.lsp4ij.settings.UserDefinedLanguageServerSettings;
+import org.eclipse.lsp4j.jsonrpc.messages.Message;
+import org.eclipse.lsp4j.jsonrpc.messages.NotificationMessage;
+import org.eclipse.lsp4j.jsonrpc.messages.RequestMessage;
+import org.eclipse.lsp4j.jsonrpc.messages.ResponseMessage;
+
+import javax.swing.tree.DefaultMutableTreeNode;
+import javax.swing.tree.TreePath;
+
+import static com.redhat.devtools.intellij.quarkus.lsp4ij.utils.UIHelper.executeInUI;
+
+public class LanguageServerExplorerLifecycleListener implements LanguageServerLifecycleListener {
+
+    private final LanguageServerExplorer explorer;

```

Comment:
private final Map<String, String> methods = new HashMap<>(10);

Reply:
would need to call `methods.clear();` in a dispose() method

Reply:
Thanks @fbricon for your feedback. I did the fix and now you should see the method and rime elapsed in the response. But there is a bug with some response which have not retrieved for java file because there are 2 ls mapped with java files. I will fix it.

Reply:
fixed